Demo/Demo/Features/CardDemoViewController.swift

```diff
@@ -0,0 +1,106 @@
+import UIKit
+
+class CardDemoViewController: FeatureBaseViewController {
+    // TODO: Validate card fields are filled out
+    // TODO: Add proper spacing / styling / max characters to card fields
+    // TODO: Animate pay button to indicate loading
+    // TODO: Disable pay button until fields are filled out
+
+    private enum Constants {
+        static let stackViewSpacing: CGFloat = 16
+        static let layoutSpacing: CGFloat = 16
```

Comment:
can we remove these two since they exists in `FeatureBaseViewController`? 
And we can make the `Constants` in `FeatureBaseViewController` `internal` so subclasses can use them.